httplib2_transport/tox.ini

```diff
@@ -0,0 +1,38 @@
+[tox]
+envlist = lint,py27,py34,py35,pypy,cover
+
+[testenv]
+deps =
+  flask
+  mock
+  pytest
+  pytest-cov
+  pytest-localserver
+  httplib2
+  {toxinidir}/..
+# Always recreate because of the relative path in deps above.
+recreate = True
+commands =
+  py.test --cov=google_auth_httplib2 --cov=tests {posargs:tests}
+
+[testenv:cover]
+basepython = python3.5
+commands =
+  py.test --cov=google_auth_httplib2 --cov=tests --cov-report= tests
+  coverage report --show-missing --fail-under=100
+deps =
+  {[testenv]deps}
+
+[testenv:lint]
+basepython = python3.5
+commands =
+  flake8 \
+    --import-order-style=google \
+    --application-import-names="google_auth_httplib2,tests" \
+    google_auth_httplib2.py tests
+  pylint --rcfile pylintrc google_auth_httplib2.py
```

Comment:
Why not just refer to `{toxinidir}/../pylintrc`? (similar for the line below)

Also, there is only one file to lint?